.github/ISSUE_TEMPLATE/Feature_Request.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I don't think this template is really necessary, but without it it would look like this, which might not be as user friendly:

![img](https://liefland.net/~zarthus/image/chrome_13749b1757493b6d.png)